index.html

```diff
@@ -131,12 +129,12 @@
             // loaded and exposed in the repl.
             scripts: [
               {
-                src      : 'http://wzrd.in/standalone/sanctuary@latest',
+                src      : '//wzrd.in/standalone/sanctuary@latest',
                 global   : 'sanctuary',
                 exposeAs : 'S'
               },
               {
-                src    : 'http://wzrd.in/standalone/ramda-fantasy@latest',
+                src    : '//wzrd.in/standalone/ramda-fantasy@latest',
```

Comment:
Yes - example: [https://wzrd.in/standalone/sanctuary@latest](https://wzrd.in/standalone/sanctuary@latest)

You can also observe it using your network panel on [this page](https://craigdallimore.github.io/repl). (Requires a click to start the repl).